src/AoeLoot.cpp

```diff
@@ -15,6 +15,155 @@
  * with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
-#include "AoeLoot.h"
+#include "Item.h"
 #include "Log.h"
+#include "Chat.h"
 #include "Config.h"
+#include "Corpse.h"
+#include "Creature.h"
+#include "Chat.h"
+#include "GameObject.h"
+#include "Group.h"
+#include "LootItemStorage.h"
+#include "LootMgr.h"
+#include "Object.h"
+#include "ObjectAccessor.h"
+#include "ObjectMgr.h"
+#include "Opcodes.h"
+#include "Player.h"
+#include "ScriptedGossip.h"
+#include "ScriptMgr.h"
+#include "WorldPacket.h"
+#include "WorldSession.h"
+
+bool AOELootEnable = 1;
+bool AOELootAnnounceModule = 1;
+
+class AoeLoot_World : public WorldScript
+{
+public:
+    AoeLoot_World() : WorldScript("AoeLoot_World") { }
+
+    void OnConfigLoad(bool reload)
+    {
+        if (!reload) {
+            // Load Configuration Settings
+            SetInitialWorldSettings();
+        }
+    }
+
+    // Load Configuration Settings
+    void SetInitialWorldSettings()
+    {
+        AOELootEnable = sConfigMgr->GetBoolDefault("AOELoot.Enable", 1);
+        AOELootAnnounceModule = sConfigMgr->GetBoolDefault("AOELoot.Announce.Enable", 1);
+    };
+};
+
+class AoeLootAnnounce : public PlayerScript
+{
+public:
+    AoeLootAnnounce() : PlayerScript("AoeLootAnnounce") {}
+    void OnLogin(Player* Player) override
+    {
+        // Announce Module
+        if (AOELootEnable)
+        {
+            if (AOELootAnnounceModule)
+            {
+                ChatHandler(Player->GetSession()).SendSysMessage("This server is running the |cff4CFF00AOE Loot Azerothcore Custom |rmodule.");
+            }
+        }
+    }
+};

```

Comment:
why need this?

Reply:
It was to test that i understood using the hooks right really. It did work and can be removed if you want. Right now i am dealing with a internet outage so my response will be slow.